config/initializers/sidekiq.rb

```diff
@@ -1,7 +1,22 @@
 Sidekiq.configure_client do |config|
   config.client_middleware do |chain|
     chain.add SidekiqLoggerMiddleware
-    # See https://github.com/mhenrixon/sidekiq-unique-jobs/blob/921aaa4efc998b102790d1f4ecce2ebac609e464/README.md#add-the-middleware
     chain.add SidekiqUniqueJobs::Middleware::Client
   end
 end
+
+Sidekiq.configure_server do |config|
+  config.client_middleware do |chain|
+    chain.add SidekiqLoggerMiddleware
+    chain.add SidekiqUniqueJobs::Middleware::Client
+  end
+
+  config.server_middleware do |chain|
+    chain.add SidekiqUniqueJobs::Middleware::Server
+  end
+
+  SidekiqUniqueJobs::Server.configure(config)
+end
+
+# SidekiqUniqueJobs recommends not testing this behaviour, our tests have previously has caused flakey builds
```

Comment:
I don't think we've actually experienced flaky builds in Publishing API so this comment might be a bit misleading for future people - might be better to instead link to: https://github.com/mhenrixon/sidekiq-unique-jobs#uniqueness ?